src/modules/shared/forms/engine/form-engine.component.ts

```diff
@@ -95,6 +95,20 @@ export class FormEngineComponent implements OnInit, OnChanges, OnDestroy {
       this.form.valueChanges.pipe(debounceTime(500)).subscribe(() => this.formChanges.emit(this.form.value))
     );
 
+    if (isPlatformBrowser(this.platformId) && this.onlyOneField) {
```

Comment:
Perfect. Can you just add some comments on what this code is for. Just to clarify when someone sees it for the first time?

Reply:
Definitely, added in the most recent commit!